### DISCUSSION TITLE
is there a way to search values in an array

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#11676] [Q&A] [answered]
This would be equivalent of `.contains` [in Rust libstd](https://doc.rust-lang.org/std/primitive.slice.html#method.contains), or `contains` (for string type) in VRL.

**Comment by spencergilbert** [comment, marked as answer]
Is [`includes`](https://vector.dev/docs/reference/vrl/functions/#includes) what you're looking for?

**Reply by tshepang** [reply]
Hehe, I've seen this one and forgot it... such unusual name. Could VRL re-use `contains` to accept array as well?

**Reply by spencergilbert** [reply]
@StephenWakely @JeanMertz thoughts on this?

**Reply by JeanMertz** [reply]
Unfortunately not, because the function signature differs too much between the two (and might diverge further in the future).

**Reply by spencergilbert** [reply]
That was my suspicion but I wanted to confirm, thanks!
